.github/workflows/e2e.yml

```
@@ -97,23 +97,23 @@ jobs:
         id: set-domain
         run: |
           echo "domain=${{ env.GITHUB_HEAD_REF_SLUG_URL }}.dashboard.saleor.rocks" >> $GITHUB_OUTPUT
-      - name: Get API_URI
-        id: api_uri
-        # Search for API_URI in PR description and use default if not defined
+      - name: Get API_URL
+        id: API_URL
+        # Search for API_URL in PR description and use default if not defined
         env:
           pull_request_body: ${{ github.event.pull_request.body }}
-          prefix: API_URI=
+          prefix: API_URL=
           pattern: (http|https)://[a-zA-Z0-9.-]+/graphql/?
-          fallback_uri: ${{ secrets.CYPRESS_API_URI }}
+          fallback_uri: ${{ secrets.CYPRESS_API_URL }}
         run: |
-          echo "custom_api_uri=$(echo "$pull_request_body" | grep -Eo "$prefix$pattern" | sed s/$prefix// | head -n 1 | { read custom_uri; if [ -z "$custom_uri" ]; then echo "$fallback_uri"; else echo "$custom_uri"; fi })" >> $GITHUB_OUTPUT
+          echo "custom_API_URL=$(echo "$pull_request_body" | grep -Eo "$prefix$pattern" | sed s/$prefix// | head -n 1 | { read custom_uri; if [ -z "$custom_uri" ]; then echo "$fallback_uri"; else echo "$custom_uri"; fi })" >> $GITHUB_OUTPUT
       - name: Checkout
         uses: actions/checkout@v2
 
       - name: Cypress run
         uses: cypress-io/github-action@v4
         env:
-          API_URI: ${{ steps.api_uri.outputs.custom_api_uri }}
+          API_URL: ${{ steps.API_URL.outputs.custom_API_URL }}
```

Next part of the mixed case:

```suggestion
          API_URL: ${{ steps.API_URL.outputs.custom_api_url }}